src/Api/ConfigureServices.cs

```diff
@@ -14,7 +14,7 @@ public static IServiceCollection AddApiServices(this IServiceCollection services
     {
         // Register services
         services.AddServices();
-        
+        services.AddHostedService<BorrowRequestService>();
```

Comment:
Should extract to a separate method for registering background services only

Reply:
okay